src/components/transport_manager/test/transport_adapter_test.cc

```diff
@@ -1240,10 +1261,12 @@ TEST_F(TransportAdapterTest, StopDevice) {
 
 TEST_F(TransportAdapterTest, TransportConfigUpdated) {
   MockTransportAdapterImpl transport_adapter(
-      NULL, NULL, NULL, last_state_, transport_manager_settings);
+      NULL, NULL, NULL, last_state_wrapper_, transport_manager_settings);
   SetDefaultExpectations(transport_adapter);
   EXPECT_CALL(transport_adapter, Restore()).WillOnce(Return(true));
   transport_adapter.Init();
+  EXPECT_CALL(transport_adapter, Restore()).WillOnce(Return(true));
+  transport_adapter.Init();
```

Comment:
Why is this init code here twice?

Reply:
@jacobkeeler Thank you. It is mistake of the last rebase. Removed in 5477ac1f48f21ba7a4312839e58907d64251d06a